============================= test session starts ==============================
platform linux -- Python 3.5.10, pytest-6.1.2, py-1.10.0, pluggy-0.13.1
rootdir: /home/james/cloud-init, configfile: tox.ini
plugins: cov-2.12.1
collected 6 items

tests/integration_tests/modules/test_package_update_upgrade_install.py::TestPackageUpdateUpgradeInstall::test_new_packages_are_installed Detected image: image_id=bionic os=ubuntu release=bionic

-------------------------------- live log setup --------------------------------
2021-11-08 16:53:04 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
Detected image: image_id=bionic os=ubuntu release=bionic
2021-11-08 16:53:05 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
Settings:
CLOUD_INIT_SOURCE=PROPOSED
COLLECT_LOGS=NEVER
EXISTING_INSTANCE_ID=None
INSTANCE_TYPE=None
KEEP_IMAGE=False
KEEP_INSTANCE=False
KEYPAIR_NAME=None
LOCAL_LOG_PATH=/tmp/cloud_init_test_logs
OPENSTACK_NETWORK=falcojr_admin_net
ORACLE_AVAILABILITY_DOMAIN=qIZq:PHX-AD-3
OS_IMAGE=bionic
PLATFORM=azure
PUBLIC_SSH_KEY=None
RUN_UNSTABLE=False
2021-11-08 16:53:05 INFO      integration_testing:clouds.py:132 Settings:
CLOUD_INIT_SOURCE=PROPOSED
COLLECT_LOGS=NEVER
EXISTING_INSTANCE_ID=None
INSTANCE_TYPE=None
KEEP_IMAGE=False
KEEP_INSTANCE=False
KEYPAIR_NAME=None
LOCAL_LOG_PATH=/tmp/cloud_init_test_logs
OPENSTACK_NETWORK=falcojr_admin_net
ORACLE_AVAILABILITY_DOMAIN=qIZq:PHX-AD-3
OS_IMAGE=bionic
PLATFORM=azure
PUBLIC_SSH_KEY=None
RUN_UNSTABLE=False
Setting up environment for azure
2021-11-08 16:53:05 INFO      integration_testing:conftest.py:156 Setting up environment for azure
Launching instance with launch_kwargs:
user_data=None
image_id=Canonical:UbuntuServer:18.04-DAILY-LTS
2021-11-08 16:53:05 INFO      integration_testing:clouds.py:171 Launching instance with launch_kwargs:
user_data=None
image_id=Canonical:UbuntuServer:18.04-DAILY-LTS
2021-11-08 16:54:36 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
2021-11-08 16:54:36 INFO      paramiko.transport:transport.py:1819 Connected (version 2.0, client OpenSSH_7.6p1)
2021-11-08 16:54:36 INFO      paramiko.transport:transport.py:1819 Authentication (publickey) successful!
2021-11-08 16:54:38 INFO      pycloudlib.instance:instance.py:175 executing: which systemctl
2021-11-08 16:54:38 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:54:38 INFO      pycloudlib.instance:instance.py:175 executing: cloud-init status --wait --long
Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7fecaa73e048>
2021-11-08 16:54:39 INFO      integration_testing:clouds.py:176 Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7fecaa73e048>
2021-11-08 16:54:39 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init --version'
cloud-init version: /usr/bin/cloud-init 21.3-1-g6803368d-0ubuntu1~18.04.4
2021-11-08 16:54:40 INFO      integration_testing:clouds.py:182 cloud-init version: /usr/bin/cloud-init 21.3-1-g6803368d-0ubuntu1~18.04.4
2021-11-08 16:54:40 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'grep serial /etc/cloud/build.info'
image serial: 20211103
2021-11-08 16:54:40 INFO      integration_testing:clouds.py:186 image serial: 20211103
Installing proposed image
2021-11-08 16:54:40 INFO      integration_testing:instances.py:152 Installing proposed image
2021-11-08 16:54:40 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'echo deb "http://archive.ubuntu.com/ubuntu $(lsb_release -sc)-proposed main" >> /etc/apt/sources.list.d/proposed.list'
2021-11-08 16:54:40 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'apt-get update -q'
2021-11-08 16:54:49 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'apt-get install -qy cloud-init'
2021-11-08 16:55:00 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init -v'
Installed cloud-init version: 21.4-0ubuntu1~18.04.1
2021-11-08 16:55:00 INFO      integration_testing:instances.py:139 Installed cloud-init version: 21.4-0ubuntu1~18.04.1
2021-11-08 16:55:00 INFO      pycloudlib.instance:instance.py:175 executing: sh -c 'sudo cloud-init clean --logs'
2021-11-08 16:55:01 INFO      pycloudlib.instance:instance.py:175 executing: sh -c 'sudo rm -rf /var/log/syslog'
2021-11-08 16:55:01 INFO      pycloudlib.instance:instance.py:175 executing: sh -c 'sudo cloud-init clean --logs'
2021-11-08 16:55:02 INFO      pycloudlib.instance:instance.py:175 executing: sh -c 'sudo rm -rf /var/log/syslog'
2021-11-08 16:55:02 INFO      pycloudlib.instance:instance.py:175 executing: sh -c 'sudo waagent -deprovision+user -force'
Created new image: /subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image
2021-11-08 16:56:04 INFO      integration_testing:instances.py:116 Created new image: /subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image
Done with environment setup
2021-11-08 16:56:44 INFO      integration_testing:conftest.py:162 Done with environment setup
Launching instance with launch_kwargs:
user_data=#cloud-config
packages:
  - sl
  - tree
package_update: true
package_upgrade: true

image_id=/subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image
2021-11-08 16:56:44 INFO      integration_testing:clouds.py:171 Launching instance with launch_kwargs:
user_data=#cloud-config
packages:
  - sl
  - tree
package_update: true
package_upgrade: true

image_id=/subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image
2021-11-08 16:57:31 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
2021-11-08 16:57:31 INFO      paramiko.transport:transport.py:1819 Connected (version 2.0, client OpenSSH_7.6p1)
2021-11-08 16:57:32 INFO      paramiko.transport:transport.py:1819 Authentication (publickey) successful!
2021-11-08 16:57:35 INFO      pycloudlib.instance:instance.py:175 executing: which systemctl
2021-11-08 16:57:35 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:36 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:37 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:38 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:39 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:40 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:41 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:42 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:43 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:44 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:45 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:46 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:47 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:49 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:50 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:51 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:52 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:53 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:54 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:55 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:56 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:57 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:58 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:57:59 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:00 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:01 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:03 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:04 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:05 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:06 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:07 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:08 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:09 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:10 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:11 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:12 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:13 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:14 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:15 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:17 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:18 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:19 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:20 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:21 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:22 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:23 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:24 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:25 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:26 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:27 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 16:58:27 INFO      pycloudlib.instance:instance.py:175 executing: cloud-init status --wait --long
Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7fecaa736128>
2021-11-08 16:58:28 INFO      integration_testing:clouds.py:176 Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7fecaa736128>
2021-11-08 16:58:28 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init --version'
cloud-init version: /usr/bin/cloud-init 21.4-0ubuntu1~18.04.1
2021-11-08 16:58:28 INFO      integration_testing:clouds.py:182 cloud-init version: /usr/bin/cloud-init 21.4-0ubuntu1~18.04.1
2021-11-08 16:58:28 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'grep serial /etc/cloud/build.info'
image serial: 20211103
2021-11-08 16:58:29 INFO      integration_testing:clouds.py:186 image serial: 20211103
-------------------------------- live log call ---------------------------------
2021-11-08 16:58:29 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'dpkg-query --show'
PASSED
tests/integration_tests/modules/test_package_update_upgrade_install.py::TestPackageUpdateUpgradeInstall::test_packages_were_updated Detected image: image_id=bionic os=ubuntu release=bionic

-------------------------------- live log setup --------------------------------
2021-11-08 16:58:29 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
-------------------------------- live log call ---------------------------------
2021-11-08 16:58:29 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'grep ^Commandline: /var/log/apt/history.log'
PASSED
tests/integration_tests/modules/test_package_update_upgrade_install.py::TestPackageUpdateUpgradeInstall::test_packages_were_upgraded Detected image: image_id=bionic os=ubuntu release=bionic

-------------------------------- live log setup --------------------------------
2021-11-08 16:58:29 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
-------------------------------- live log call ---------------------------------
2021-11-08 16:58:29 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cat /var/log/cloud-init-output.log'
PASSED
tests/integration_tests/modules/test_package_update_upgrade_install.py::TestPackageUpdateUpgradeInstall::test_new_packages_are_installed Detected image: image_id=bionic os=ubuntu release=bionic

-------------------------------- live log setup --------------------------------
2021-11-08 16:59:09 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
Launching instance with launch_kwargs:
user_data=#cloud-config
packages:
  - sl
  - tree
package_update: true
package_upgrade: true

image_id=/subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image
2021-11-08 16:59:09 INFO      integration_testing:clouds.py:171 Launching instance with launch_kwargs:
user_data=#cloud-config
packages:
  - sl
  - tree
package_update: true
package_upgrade: true

image_id=/subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image
2021-11-08 16:59:56 INFO      pycloudlib.instance:instance.py:175 executing: sh -c whoami
2021-11-08 16:59:56 INFO      paramiko.transport:transport.py:1819 Connected (version 2.0, client OpenSSH_7.6p1)
2021-11-08 16:59:56 INFO      paramiko.transport:transport.py:1819 Authentication (publickey) successful!
2021-11-08 16:59:58 INFO      pycloudlib.instance:instance.py:175 executing: which systemctl
2021-11-08 16:59:59 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:00 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:01 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:02 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:03 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:04 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:05 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:06 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:07 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:08 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:09 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:10 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:11 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:12 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:14 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:15 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:16 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:17 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:18 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:19 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:20 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:21 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:22 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:23 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:24 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:25 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:26 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:27 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:28 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:30 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:31 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:32 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:33 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:34 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:35 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:36 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:37 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:38 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:39 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:40 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:41 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:42 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:43 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:45 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:46 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:47 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:48 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:49 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:50 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:51 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:52 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:53 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:54 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:55 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:56 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:57 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:58 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:00:59 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:01 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:02 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:03 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:04 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:05 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:06 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:07 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:08 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:09 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:10 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:11 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:12 INFO      pycloudlib.instance:instance.py:175 executing: systemctl is-active cloud-init.target
2021-11-08 17:01:12 INFO      pycloudlib.instance:instance.py:175 executing: cloud-init status --wait --long
Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7feca95a1278>
2021-11-08 17:01:13 INFO      integration_testing:clouds.py:176 Launched instance: <pycloudlib.azure.instance.AzureInstance object at 0x7feca95a1278>
2021-11-08 17:01:13 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cloud-init --version'
cloud-init version: /usr/bin/cloud-init 21.4-0ubuntu1~18.04.1
2021-11-08 17:01:14 INFO      integration_testing:clouds.py:182 cloud-init version: /usr/bin/cloud-init 21.4-0ubuntu1~18.04.1
2021-11-08 17:01:14 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'grep serial /etc/cloud/build.info'
image serial: 20211103
2021-11-08 17:01:14 INFO      integration_testing:clouds.py:186 image serial: 20211103
-------------------------------- live log call ---------------------------------
2021-11-08 17:01:14 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'dpkg-query --show'
PASSED
tests/integration_tests/modules/test_package_update_upgrade_install.py::TestPackageUpdateUpgradeInstall::test_packages_were_updated Detected image: image_id=bionic os=ubuntu release=bionic

-------------------------------- live log setup --------------------------------
2021-11-08 17:01:14 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
-------------------------------- live log call ---------------------------------
2021-11-08 17:01:14 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'grep ^Commandline: /var/log/apt/history.log'
PASSED
tests/integration_tests/modules/test_package_update_upgrade_install.py::TestPackageUpdateUpgradeInstall::test_packages_were_upgraded Detected image: image_id=bionic os=ubuntu release=bionic

-------------------------------- live log setup --------------------------------
2021-11-08 17:01:14 INFO      integration_testing:clouds.py:91 Detected image: image_id=bionic os=ubuntu release=bionic
-------------------------------- live log call ---------------------------------
2021-11-08 17:01:14 INFO      pycloudlib.instance:instance.py:175 executing: sudo -- sh -c 'cat /var/log/cloud-init-output.log'
PASSEDDeleting snapshot image created for this testrun: /subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image

------------------------------ live log teardown -------------------------------
2021-11-08 17:01:55 INFO      integration_testing:clouds.py:207 Deleting snapshot image created for this testrun: /subscriptions/36c65514-b88a-44f1-a44a-795c5b6fb73c/resourceGroups/azure-integration-test-1108-105304-rg/providers/Microsoft.Compute/images/azure-integration-test-1108-105305-image


=============================== warnings summary ===============================
../envs/cloud-init/lib/python3.5/site-packages/jwt/utils.py:8
  /home/james/envs/cloud-init/lib/python3.5/site-packages/jwt/utils.py:8: CryptographyDeprecationWarning: Python 3.5 support will be dropped in the next release of cryptography. Please upgrade your Python.
    from cryptography.hazmat.primitives.asymmetric.utils import (

tests/integration_tests/modules/test_package_update_upgrade_install.py: 50 warnings
  /home/james/envs/cloud-init/lib/python3.5/site-packages/urllib3/util/retry.py:440: DeprecationWarning: Using 'method_whitelist' with Retry is deprecated and will be removed in v2.0. Use 'allowed_methods' instead
    DeprecationWarning,

-- Docs: https://docs.pytest.org/en/stable/warnings.html
================== 6 passed, 51 warnings in 531.92s (0:08:51) ==================
